partner_firstname/views/res_partner.xml

```diff
@@ -0,0 +1,79 @@
+<odoo>
+    <record id="view_partner_form" model="ir.ui.view">
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form" />
+        <field name="priority" eval="99" />
```

Comment:
Why do you need a priority?

Reply:
Pre-commit was throwing a warning on this. Inheritance without a priority is dangerous.

Reply:
No, replace without priority is dangerous. And that's aligned with what I said that using replace is a bad practice. Please, remove the priority